src/main/java/lotto/domain/LottoTickets.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTickets {
+    private List<LottoTicket> lottoTickets;
+
+    public LottoTickets(List<LottoTicket> lottoTickets){
+        this.lottoTickets = lottoTickets;
+    }
+
+    public List<LottoTicket> list(){
+        return new ArrayList<>(lottoTickets);
+    }
```

Comment:
사실상 getter와 다를게 없네요! ```list()``` 라는 이름이 어울릴까요? 
```list()``` 이름만 봤을때는 해당 메서드가 어떤 일을 하는지 잘 모르겠네요.
```suggestion
    public List<LottoTicket> getLottoTickets() {
        return new ArrayList<>(lottoTickets);
    }
```

Reply:
코드를 작성할 때는 LottoTickets가 LottoTicket의 일급 컬렉션이므로 list()라는 이름이 LottoTicket 리스트를 가져온다라는 뜻을 포함한다고 생각했습니다.
근데 LottoTickets가 어떤 역할을 하는지 모르는 상태에서는 LottoTicket의 리스트인지 다른 리스트인지 파악하기 힘들거 같네요
LottoNumber를 감싸는 LottoTicket에도 같은 메소드가 있어서 이 부분도 getLottoNumbers()로 같이 바꾸어 주려 합니다.